usr/initiator.c

```diff
@@ -260,6 +260,9 @@ __session_conn_create(iscsi_session_t *session, int cid)
 
 	/* set session reconnection retry max */
 	session->reopen_max = rec->session.reopen_max;
+	session->reopen_log_freq = rec->session.reopen_log_freq;
+	if (session->reopen_log_freq == 0)
+		session->reopen_log_freq = 1;
```

Comment:
Perhaps you should just set it to "1" if it is less than 1? So we can handle negative numbers (and ignore them)?

Also, you should mention in iscsid.conf that numbers below 1 will be treated as "1". Lastly, seems like there should be a "maximum"? What if they set it to 10,000? Isn't that really "just turn it off"? Seems like, at the least, some maximum like "100" would be reasonable. And perhaps we should allow "0" to mean "just turn it off"? What do you think?